project/apps/account/src/app/mongo/mongo.config.ts

```diff
@@ -13,27 +13,27 @@ const dbValidationSchema = Joi.object({
   authBase: Joi.string().required(),
 });
 
-const validateMongoConfig = (config: MongoConfig) => {
-  const { error } = dbValidationSchema.validate(config);
+const validateMongoConfig = (config: MongoConfig): MongoConfig => {
```

Comment:
Странно) Функция для валидации принимает на вход MongoConfig и возвращает MongoConfig) Что ж она тогда валидирует, если данные и так должны приходить в правильном виде) Я бы такой тип указал
```suggestion
const validateMongoConfig = (config: Record<string, any>): MongoConfig => {
```